src/frontend/src/btc/components/core/BtcWalletAddress.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import { isNetworkIdBTCMainnet } from '$icp/utils/ic-send.utils.js';
+	import Copy from '$lib/components/ui/Copy.svelte';
+	import { btcAddressMainnet, btcAddressTestnet } from '$lib/derived/address.derived';
+	import { networkId } from '$lib/derived/network.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { shortenWithMiddleEllipsis } from '$lib/utils/format.utils';
+
+	// Regtest and Testnet BTC wallets have the same address
+	const address = isNetworkIdBTCMainnet($networkId) ? btcAddressMainnet : btcAddressTestnet;
```

Comment:
Same comment as in previous PR, should we default to mainnet?

Reply:
Done.